src/main/resources/conf/chains/pottery-genesis.json

```diff
@@ -0,0 +1,12 @@
+{
+  "difficulty": "0x20000",

```

Comment:
I think a lower difficulty is possible: https://github.com/input-output-hk/mantis/blob/develop/src/rpcTest/resources/privateNetConfig/conf/rpc-test-private-genesis.json#L5

Reply:
good idea...